lib/basedriver/commands/find.js

```diff
@@ -62,6 +68,74 @@ commands.findElementsFromElement = async function (strategy, selector, elementId
   return await this.findElOrElsWithProcessing(strategy, selector, true, elementId);
 };
 
+/**
+ * Find an element using a custom plugin specified by the customFindModule cap.
+ *
+ * @param {string} selector - the selector which the plugin will use to find
+ * elements
+ * @param {boolean} multiple - whether we want one element or multiple
+ *
+ * @returns {WebElement} - WebDriver element or list of elements
+ */
+commands.findByCustom = async function (selector, multiple) {
+  if (!this.opts.customFindModule) {
```

Comment:
do we allow only one such module to be set at the same time or many of them?

Reply:
the way the code is written now, just one module. but see the discussion we're having about whether we want something else.